core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/workflow/WorkflowStep.java

```diff
@@ -0,0 +1,36 @@
+package uk.ac.bbsrc.tgac.miso.core.data.workflow;
+
+import uk.ac.bbsrc.tgac.miso.core.data.workflow.impl.IntegerProgressStep;
+import uk.ac.bbsrc.tgac.miso.core.data.workflow.impl.PoolProgressStep;
+import uk.ac.bbsrc.tgac.miso.core.data.workflow.impl.SampleProgressStep;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'uk.ac.bbsrc.tgac.miso.core.data.workflow.impl.SampleProgressStep'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)